meson.build

```diff
@@ -1,6 +1,6 @@
 
 project('conmon', 'c',
-        version : '0.1',
+        version : '0.1.0',
```

Comment:
Is there any way to get this in a different way so we don't need to define it twice?

Something like
```
version = run_command(
    'cat', files('VERSION'),
).stdout().strip()


project('conmon', 'c',
        version : version,
        license : 'Apache-2.0',
        default_options : [
                'c_std=c99',
        ],
        meson_version : '>= 0.46',
)
...
```

Reply:
That is what googling of mesa got me.

@saschagrunert PTAL

Reply:
good call, I modified it (and tested it)